cxx-qt-gen/src/syntax/attribute.rs

```diff
@@ -136,4 +186,48 @@ mod tests {
         assert_eq!(attribute_tokens_to_list(&module.attrs[4]).unwrap()[1], "B");
         assert_eq!(attribute_tokens_to_list(&module.attrs[4]).unwrap()[2], "C");
     }
+
+    #[test]
+    fn test_attribute_tokens_to_map() {
+        let module: ItemMod = tokens_to_syn(quote! {
```

Comment:
need to add `#[attribute()]` as a test case to check that an empty map is considered (and check if the list tests has it too).

Reply:
Fixed and added some tests in mod.rs for namespace too.